sites/docs/src/routes/+page.svelte

```diff
@@ -16,8 +16,7 @@
 <main class="flex place-items-center justify-center w-full px-4 min-h-[calc(100vh-3.5rem)] pb-10">
 	<div class="max-w-xl w-full">
 		<div class="flex flex-col gap-5 justify-center place-items-center w-full">
-			<section class="h-80"></section>
-			<div class="flex place-items-center gap-4 justify-center w-full">
+			<div class="flex place-items-center gap-4 justify-center w-full pt-40">
```

Comment:
The padding here is intentional but probably not the correct way to do it.

Ideally the hero section (logo, snippet, buttons etc.) would be centered relative to the screen but the youtube video would just peek a bit at the bottom so that you know there is more scrollable content. 

Right now it looks like this on desktop:
![image](https://github.com/user-attachments/assets/b8127316-21d5-47e9-8b70-09b9b82edc5b)

I think maybe the correct effect could be achieved by just centering the hero section and then adding a negative margin on the iframe.

If you don't have time I can commit to this PR just lmk!